src/interfaces/interfaces.ts

```diff
@@ -63,6 +63,11 @@ export interface IToken {
     logoURI: string
 }
 
+export type ITokenPriceData = {
+    chainId: string
+    price: number
+} & IToken
+
```

Comment:
nice!

Concern: This interface is used just on the request flows - wdyt about using that in other flows as well?

Reply:
I made it thinking that can be used in other flows too, so I agree